pkg/provider/akeyless/akeyless.go

```diff
@@ -256,7 +257,27 @@ func (a *Akeyless) GetSecret(ctx context.Context, ref esv1beta1.ExternalSecretDa
 	if err != nil {
 		return nil, err
 	}
-	return []byte(value), nil
+
+	if ref.Property == "" {
+		if value != "" {
+			return []byte(value), nil
+		}
+		return nil, fmt.Errorf("invalid value received, found no value string : %s", ref.Key)
+	}
+	// We need to search if a given key with a . exists before using gjson operations.
+	idx := strings.Index(ref.Property, ".")
+	if idx > -1 {
+		refProperty := strings.ReplaceAll(ref.Property, ".", "\\.")
```

Comment:
why do we need the "replace" in this case? according to the documentation,  it should work out of the box (https://github.com/tidwall/gjson#get-a-value).

@charan986  did you manually test it?

Reply:
@renanaAkeyless - I manually tested all the JSON use cases that needs to be fetched using these changes

Below is the JSON key "b.cd" test done for the above 
{
  "abc": "mega-value \n",
  "b.cd": "Wahoo!"
}

Also for reference, We followed how aws was trying to fetch the JSON key values

https://github.com/external-secrets/external-secrets/blob/main/pkg/provider/aws/secretsmanager/secretsmanager.go#L411-L417